wdl-engine/src/backend/docker.rs

```diff
@@ -66,19 +66,19 @@ use crate::v1::memory;
 const INITIAL_EXPECTED_NAMES: usize = 1000;
 
 /// The root guest path for inputs.
-const GUEST_INPUTS_DIR: &str = "/mnt/inputs";
+const GUEST_INPUTS_DIR: &str = "/mnt/task/inputs";
 
 /// The guest working directory.
-const GUEST_WORK_DIR: &str = "/mnt/work";
+const GUEST_WORK_DIR: &str = "/mnt/task/work";
 
 /// The guest path for the command file.
-const GUEST_COMMAND_PATH: &str = "/mnt/command";
+const GUEST_COMMAND_PATH: &str = "/mnt/task/command";
 
 /// The path to the container's stdout.
-const GUEST_STDOUT_PATH: &str = "/var/log/stdout";
+const GUEST_STDOUT_PATH: &str = "/mnt/task/stdout";
```

Comment:
In putting it in the same directory, is there a chance this could get clobbered if the task writes to a file called `stdout`?

Reply:
I see now the other comment. I actually preferred `/var/log/stdout` and `/var/log/stderr` since those are the canonical place for logs on Linux. I'm fine with the other location (just my opinion).